packages/grafana-toolkit/src/config/utils/pluginValidation.ts

```diff
@@ -36,3 +36,12 @@ export const getPluginJson = (path: string): PluginMeta => {
 
   return pluginJson as PluginMeta;
 };
+
+export const isValidRootUrl = (rootUrl: string): boolean => {
+  try {
+    new URL(rootUrl);
+    return true;
+  } catch (err) {
+    return false;
+  }
```

Comment:
nit: given how generic this is maybe we drop root from the naming?

```suggestion
export const isValidUrl = (url: string): boolean => {
  try {
    new URL(url);
    return true;
  } catch (err) {
    return false;
  }
```

Reply:
I agree with what Jack said, it would probably be better to call it a bit more generic.

Or we could do the following as well to make the call-site a little bit simpler:
```javascript

// pluginValidation.ts
export const assertRootUrlIsValid = (url: string) => {
  try {
    new URL(url);
  } catch (err) {
    throw new Error(`${rootUrl} is not a valid URL`);
  }
}

// ...

// plugin.ci.ts
rootUrls.forEach(assertRootUrlIsValid);


```